2/contrib/jenkins/install-jenkins-core-plugins.sh

```diff
@@ -18,8 +18,12 @@ if [[ "${INSTALL_JENKINS_VIA_RPMS}" == "false" ]]; then
       rm -fr /var/cache/yum/x86_64/7Server/*
       rm -fr /var/cache/yum/x86_64/7Server/ # Clean yum cache otherwise, it will fail if --disablerepos are specified
     fi
-    yum -y $YUM_FLAGS --setopt=tsflags=nodocs --disableplugin=subscription-manager install jenkins-2.289.2
-    rpm -V jenkins-2.289.2
+    # Since the recent LTS jenkins update we need to install the 'daemonize' package
+    # which is only available in EPEL, so enable it here
+    yum -y --setopt=tsflags=nodocs --disableplugin=subscription-manager install \
+	    https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+    yum -y $YUM_FLAGS --setopt=tsflags=nodocs --disableplugin=subscription-manager install jenkins-2.303.3
+    rpm -V jenkins-2.303.3
```

Comment:
@akram I am pretty certain this will break our downstream build with ART. Do we need to coordinate with the test platform team and/or ART to address this?

Reply:
yeah this has come up before ... we have to use rhel/ubi packages not fedora 

as @akram noted in https://github.com/openshift/jenkins/pull/1332#issuecomment-961316852

including reaching out to the ART team in their channel to find out where in rhel land the `daemonize` package lives .... they have been helpful with that in the past

Reply:
I don't see any such package internally, only Fedora and EPEL (which, as you noted, is not acceptable for OCP builds).

Reply:
@adambkaplan ART is setting the INSTALL_JENKINS_VIA_RPMS env var in osbs/brew, so they are passing here:
https://github.com/openshift/jenkins/blob/663a4b706c50aae7dddc6d11d7bdfc79ff4af7a6/2/contrib/jenkins/install-jenkins-core-plugins.sh#L29

we should be ok

Reply:
@yselkowitz so for our downstream build we make our own RPM spec that has nothing to do with what upstream does. We don't need `daemonize`: https://pkgs.devel.redhat.com/cgit/rpms/jenkins/tree/jenkins.spec?h=rhaos-4.9-rhel-8 (needs VPN).

@alicerum please update this PR so that the enablement of EPEL happens inside the `[[ "${INSTALL_JENKINS_VIA_RPMS}" == "false" ]];` if block.

Reply:
@adambkaplan it already is inside this `if` block, which begins earlier in the code. in the current state `EPEL` should not be enabled for our own downstream build.

Reply:
ack - misread the code then